Configuration/DataProcessing/python/Impl/AlCaPPS_Run3.py

```diff
@@ -11,11 +11,12 @@
 import sys
 
 from Configuration.DataProcessing.Scenario import *
+from Configuration.DataProcessing.Impl.AlCa import AlCa
 from Configuration.DataProcessing.Utils import stepALCAPRODUCER,dqmIOSource,harvestingMode,dictIO,gtNameAndConnect,addMonitoring
 from Configuration.Eras.Era_Run3_cff import Run3
 import FWCore.ParameterSet.Config as cms
 
-class AlCaPPS_Run3(Scenario):
+class AlCaPPS_Run3(AlCa):
     def __init__(self):
         Scenario.__init__(self)
```

Comment:
maybe I'm wrong, but I would think that this should be the AlCa class being initialised here, instead of the Scenario one.

Reply:
Hmm probably that would be better. But it does work like this too... My understanding is that it initializes the `Scenario`  but inherits from `AlCa`, so when the harvesting function was missing earlier now it's included.  The only missing part so far was this harvesting function, the rest worked with `Scenario` too. So that's why I think this works.

Reply:
Given that it works, I'd prefer to keep it like this. Do you agree?

Reply:
Yes, I agree.